### PR TITLE
Add projects.close endpoint

### PR DESCRIPTION
**Pull request by joaoinacio**

### CODE REVIEW

**Review by ArnaudWeyts — DISMISSED** [review]

**Review by mikeverf — DISMISSED** [review]

File: src/07-projects/projects.apib [hunk 1]
```diff
@@ -211,6 +211,18 @@ Update a project.
 + Response 204 (application/json)
 
 
+### projects.close [POST /projects.close]
+
+Closes a project, and all its phases
```
Comment (adognameddog) [inline src/07-projects/projects.apib]: I think we should also mention it closes tasks, but not meetings.

**Review by adognameddog — DISMISSED** [review]

**Review by mikeverf — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -3486,6 +3487,18 @@ Update a project.
...
+### projects.close [POST /projects.close]
+
+Closes a project, all its phases, and all tasks within each phase (but not meetings).
+
++ Request (application/json)
+
+    + Attributes (object)
+        + id: `01548b10-4932-4a18-8a89-005ad09db2c8` (string, required)
+
++ Response 204 (application/json)
```
Comment (tiagobastos) [inline apiary.apib:3499]: `204 - No content` has no content so the `application/json` response doesn't apply right?

Comment (joaoinacio) [reply apiary.apib:3499]: > `204 - No content` has no content so the `application/json` response doesn't apply right?

Good point, I think you may be right (the content-type shouldn't be on the response).
OTOH I copy-pasted it, so would probably make sense to do a follow-up PR fixing this on every endpoint.

Comment (tiagobastos) [reply apiary.apib:3499]: Good with a follow-up.

**Review by tiagobastos — APPROVED** [review]

**Review by adognameddog — APPROVED** [review]
:white_check_mark: Manually checked

**Review by ArnaudWeyts — APPROVED** [review]

**Pull request merged**